test/Sdk.Generator.Tests/FunctionMetadataProviderGeneratorTests/StorageBindingTests.cs

```diff
@@ -236,7 +236,7 @@ public Task<ImmutableArray<IFunctionMetadata>> GetFunctionMetadataAsync(string d
                 name = 'blob',
                 type = 'BlobTrigger',
                 direction = 'In',
-                blobPath = 'container2/%file%',
+                path = 'container2/%file%',
```

Comment:
This test has been changed to reflect that blob triggers should have an arg name of "path", not "blobPath". In line 201 above for the Blob output binding, we still keep the arg name "blobPath". 

[Blob bindings classess](https://github.com/Azure/azure-functions-dotnet-worker/tree/main/extensions/Worker.Extensions.Storage.Blobs/src) for reference.